projects/storefrontstyles/scss/components/myaccount/_my-interests.scss

```diff
@@ -0,0 +1,222 @@
+%cx-my-interests {
+  .cx-product-interests-title {
+    margin: 0 auto;
+  }
+  .cx-product-interests {
+    @include media-breakpoint-down(sm) {
+      max-width: 1%;
+      padding: 0;
+    }
+  }
+  .cx-product-interests-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-width: 0 0 1px 0;
+    border-color: var(--cx-color-light);
+    border-style: solid;
+
+    tr {
+      width: 100%;
+
+      @include media-breakpoint-down(sm) {
+        border-width: 1px 0 0 0;
+        border-color: var(--cx-color-light);
+        border-style: solid;
+
+        &:first-child {
+          border-width: 1px 0 0 0;
+          padding: 1.25rem 0 0 0;
+        }
+      }
+    }
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      text-align: left;
+
+      &:first-child {
+        text-align: center;
+      }
+    }
+
+    td {
+      width: 25%;
+      padding: 1.625rem 0;
+
+      &:first-child {
+        width: 10%;
+        padding-right: 1.25rem;
+        padding-left: 1.25rem;
+      }
+      &:last-child {
+        justify-content: flex-end;
+      }
+
+      @include media-breakpoint-up(md) {
+        &:last-child {
+          text-align: center;
+        }
+      }
+
+      .cx-name {
+        a {
+          color: $dark;
+          font-weight: var(--cx-font-weight-bold);
+        }
+      }
+
+      .cx-actions {
+        button.link {
+          font-weight: var(--cx-font-weight-bold);
+          font-size: var(--cx-font-size, 0.875rem);
+        }
+      }
+
+      .cx-code {
+        font-size: var(--cx-font-size, 0.875rem);
+        font-weight: var(--cx-font-weight-normal);
+        line-height: var(--cx-line-height, 1.2222222222);
```

Comment:
we have discontinued the use of css variables for everything except colors, the line-height value can be assigned directly, also for fonts we're using the `type()` mixin, as per table below. Therefore font-size and font-weight can be replaced in all instance by the proper mixin.

![Screen Shot 2019-09-27 at 10 59 00 AM](https://user-images.githubusercontent.com/15113219/70110350-ac125780-161c-11ea-9ba9-d926265e669f.png)

Reply:
update.